custom_components/easee/services.py

```diff
@@ -295,7 +307,7 @@ async def charger_execute_set_charging_cost(call):
         currency = call.data.get(ATTR_COST_CURRENCY)
         vat = call.data.get(ATTR_COST_VAT)
 
-        _LOGGER.debug("execute_service:" + str(call.data))
+        _LOGGER.debug("execute_service: " + str(call.service) + " " + str(call.data))
```

Comment:
Follow HA code guidelines
```suggestion
        _LOGGER.debug("execute_service: %s %s", str(call.service), str(call.data))
```